src/animals/Antelope.js

```diff
@@ -0,0 +1,33 @@
+import { Animal } from '../organisms/Animal';
+import { tryWithChance } from '../utilities/tryWithChance';
+
+export class Antelope extends Animal {
+  constructor(positionY, positionX, age = 0) {
+    super(4, 4, positionY, positionX, age);
+  }
+
+  getAvailableDirections(board) {
+    const directions = [];
+    for (let directionY = -2; directionY <= 2; directionY++) {
+      for (let directionX = -2; directionX <= 2; directionX++) {
+        if (!(directionX === 0 && directionY === 0)) {
```

Comment:
Simplify this `if` statement. Avoid using `!` to negate multiple booleans at once like that. It's difficult to read.